charts/clusternet-controller-manager/README.md

```diff
@@ -0,0 +1,83 @@
+
+### Uninstalling the Chart
+
+To uninstall/delete the `clusternet-controller-manager` deployment:
+
+```bash
+helm delete clusternet-controller-manager -n clusternet-system
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+### Parameters
+
+#### Common parameters
+
+| Name                | Description                                        | Value |
+| ------------------- | -------------------------------------------------- | ----- |
+| `kubeVersion`       | Override Kubernetes version                        | `""`  |
+| `nameOverride`      | String to partially override common.names.fullname | `""`  |
+| `fullnameOverride`  | String to fully override common.names.fullname     | `""`  |
+| `commonLabels`      | Labels to add to all deployed objects              | `{}`  |
+| `commonAnnotations` | Annotations to add to all deployed objects         | `{}`  |
+
+#### Exposure parameters
+
+| Name                       | Description                                                                               | Value                                                                                            |
+|----------------------------|-------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------|
+| `replicaCount`             | Specify number of clusternet-controller-manager replicas                                  | `3`                                                                                              |
+| `serviceAccount.name`      | The name of the ServiceAccount to create                                                  | `"clusternet-controller-manager"`                                                                |
+| `securePort`               | Port where clusternet-controller-manager will be running                                  | `443`                                                                                            |
+| `controller`               | Controller a list of controllers to enable                                                | `"*"`                                                                                              |
+| `image.registry`           | clusternet-controller-manager image registry                                              | `ghcr.io`                                                                                        |
+| `image.repository`         | clusternet-controller-manager image repository                                            | `clusternet/clusternet-controller-manager`                                                       |
+| `image.tag`                | clusternet-controller-manager image tag (immutable tags are recommended)                  | `v0.14.0`                                                                                        |
```

Comment:
Please update the values in this table, such as `image.tag`, `feature-gates`, etc.